### DISCUSSION TITLE
Mod pack gets stuck updating

### DISCUSSION THREAD

**Discussion by theflopmeister** [modrinth/.github#299] [Other discussions]
If you try to update a mod pack while that instance is running, the update will fail, and the instance will be unlaunchable. It doesn't throw any errors or give any visible warnings that the instance is running. Once this has happened, that instance will be "installing" indefinitely. 
![image](https://github.com/user-attachments/assets/e171159e-6f3d-4d31-b55a-ade94ab72976)
Clearing the app cache, restarting modrinth, or restarting the PC doesn't fix the issue

The only ways I have found to fix this issue are either to uninstall and reinstall Modrinth or open the app.db file in an editor and change the Install_stage field for that instance from "pack_installing" to "installed".

Steps to reproduce:

1. launch an outdated modpack
2. With the game still running, open the mod pack and update it
3. close the game
4. the instance should now say "installing" and be unlaunchable

**Comment by HeroOfDungeon** [comment]
I have the same problem and have found another way to fix it:
1. export the modpack with all possible data
2. import the modpack you just exported to make sure everything is ok
3. delete the old modpack

**Info: The modpack is no longer a modpack of the original creator, so if the creator adds a new mod, you will not get it (and the modpack image will be deleted)**
